### ISSUE TITLE
Android Downloading Games and Catalogue.xml issues

### ISSUE THREAD

**Issue by Terrapin2190** [gerstrong/Commander-Genius#331]
I left a comment in the Play Store about issues I'm having. After installing CG and running the game, things seem to run okay if choosing to download only the Game Manager. When opting to download Keen 1, Keen 4, and/or the HQ pack, I receive an error at the top of the screen showing that CG is attempting to download files from Sourceforge, but "Failed to connect to server" while downloading hqpv26.zip.

From there, the initialization screen (with the big CG logo) hangs for a few seconds and force closes itself or upon any input from menu or back keys. Which makes it impossible to play the game or get to Game Manager unless I go into initialization options and select Downloads and un-check boxes for Keen 1, Keen 4, and HQ pack.

Also, while in Game Manager, I am able to download games via the "More +" button, but after downloading any game, I receive a message about CG not being able to find catalogue.xml. The "More +" button tgen disappears from Game Manager and I have to exit the game and re-enter in order to download more games.

**Comment by gerstrong** [comment]
Yes, the reason for that is the search path system getting messed because Android has Special permission system and sandboxing. I'm working on a new Framework for CG. When finished that problem will go away.Am 24.02.2018 12:42 vorm. schrieb Terrapin2190 <notifications@github.com>:I left a comment in the Play Store about issues I'm having. After installing CG and running the game, things seem to run okay if choosing to download only the Game Manager. When opting to download Keen 1, Keen 4, and/or the HQ pack, I receive an error at the top of the screen showing that CG is attempting to download files from Sourceforge, but "Failed to connect to server" while downloading hqpv26.zip.
From there, the initialization screen (with the big CG logo) hangs for a few seconds and force closes itself or upon any input from menu or back keys. Which makes it impossible to play the game or get to Game Manager unless I go into initialization options and select Downloads and un-check boxes for Keen 1, Keen 4, and HQ pack.
Also, while in Game Manager, I am able to download games via the "More +" button, but after downloading any game, I receive a message about CG not being able to find catalogue.xml. The "More +" button tgen disappears from Game Manager and I have to exit the game and re-enter in order to download more games.
